### PR TITLE
plotst:0.1.0

### PR DESCRIPTION
**Pull request by Pegacraft**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

**Description:**
Plotst is a package people can use to create a few types of different plots and graphs. It can be useful to display data in a straight forward way.

It supports:
- Scatter plots
- Graph charts
- Histograms
- Bar charts
- Pie charts
- Overlaying plots/charts

### CODE REVIEW

File: packages/preview/plotst-0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+If you need some example code, check out `main.typ` in the `examples` folder. It also includes a compiled version of the current `main.typ`
+
+## Examples:
+
+All these images were created using the `main.typ`
+
+### Scatter plots
+
+```js
+    // Plot 1:
```
Comment (laurmaedje) [inline packages/preview/plotst-0.1.0/README.md:36]: There seems to be some superfluous indent here. Would be great if the examples were indented consistently with 2 or 4 spaces. Ideally 2 as that is the "official" style.

File: packages/preview/plotst-0.1.0/example/main.typ [hunk 2]
```diff
@@ -0,0 +1,212 @@
+#import "/lib.typ": *
```
Comment (laurmaedje) [inline packages/preview/plotst-0.1.0/example/main.typ:1]: I'm a bit on the fence with this one. To make this example copy-pasteable it should use the `@preview` import. That makes it difficult to test locally though, of course.

**Comment by laurmaedje** [comment]
There is one more thing I am unsure about: This package contains a lot of auxiliary files (examples, docs, etc.) When somebody imports this packages, these will all be downloaded as a tar.gz and be made available to the compiler. The archive for this package would be 451KB even though the actual source files are probably a few KB. There is the question whether the images and/or docs files should only be part of the external repository and linked to.

**Comment by Pegacraft** [comment]
> There is one more thing I am unsure about: This package contains a lot of auxiliary files (examples, docs, etc.) When somebody imports this packages, these will all be downloaded as a tar.gz and be made available to the compiler. The archive for this package would be 451KB even though the actual source files are probably a few KB. There is the question whether the images and/or docs files should only be part of the external repository and linked to.

You are absolutely right. I can remove all the auxiliary stuff and just link to my repo. That would leave the plain code at 9 files, if I counted correctly ^^

File: packages/preview/plotst-0.1.0/example/main.typ (hunk 2 above)
Comment (Pegacraft) [reply packages/preview/plotst-0.1.0/example/main.typ]: I could put both in and comment one with "for local testing" or something. On my repo that is. I removed the file here

File: packages/preview/plotst-0.1.0/README.md (hunk 1 above)
Comment (Pegacraft) [reply packages/preview/plotst-0.1.0/README.md]: Changed the readme to have consistent indentation. Now it should be fine

File: packages/preview/plotst-0.1.0/example/main.typ (hunk 2 above)
Comment (laurmaedje) [reply packages/preview/plotst-0.1.0/example/main.typ]: Up to you then!

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**